lib/mappers/nypl_mapper.py

```diff
@@ -300,6 +300,12 @@ def map_collection_and_relation(self):
 
         self.update_source_resource(ret_dict)
 
+    def map_edm_rights(self):
+        prop = "rightsStatementURI"
+
+        if exists(self.provider_data, prop):
+            self.mapped_data.update({"rights": getprop(self.provider_data, prop)})
```

Comment:
That is what I @guegueng told me and I just double checked it against the MAPv3 schema https://github.com/dpla/ingestion/blob/develop/lib/mapv3.py#L91